contracts/wrapped-assets/shiba/WSSLPUserProxy.sol

```diff
@@ -44,14 +43,14 @@ contract WSSLPUserProxy {
         require(user == address(0), "Unit Protocol Wrapped Assets: ALREADY_INITIALIZED");
 
         user = _user;
```

Comment:
note: now `user` is using only as marker that contract was initialised and a way to know user by proxy.